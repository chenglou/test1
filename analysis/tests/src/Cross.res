let crossRef = References.x
//               ^ref