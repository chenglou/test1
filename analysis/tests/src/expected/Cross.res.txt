References tests/src/Cross.res 0:17
[

]

